swiftide/src/transformers/metadata_qa_code.rs

```diff
@@ -7,6 +7,8 @@ use anyhow::Result;
 use async_trait::async_trait;
 use indoc::indoc;
 
+pub const NAME: &str = "Questions and Answers (code)";
```

Comment:
1. I wanted to store somewhere vector/metadata names
2. QA Metadata Code and Text had duplicated names. I changed them to avoid problems downstream.

Reply:
A disclaimer comment.
Resolving it to calm down PR merging rules.